testFile

```diff
@@ -0,0 +1 @@
chebi","CHEBI:59698"],["chebi","CHEBI:26078"],["chebi","CHEBI:46787"]]],["Standard Name","trihydrogen tetraoxophosphate(3-)"]]}},{"data":{"id":"SmallMolecule_b57f4bc630f82723a2fb03971309424c_BiochemicalReaction_9b1c77d3f3c80949b9bb522b765a72d4_LEFT_TO_RIGHT","class":"simple chemical","label":"NADH","parent":"","clonemarker":true,"stateVariables":[],"unitsOfInformation":[],"bbox":{"x":419.70395,"y":503.83844,"w":45,"h":15},"parsedMetadata":[["Type","bp:SmallMolecule"],["Data Source","http://pathwaycommons.org/pc2/kegg"],["Display Name","NADH"],["Comment","REPLACED http://www.ra.cs.uni-tuebingen.de/software/KEGGtranslator/cpdC00004"],["Names",[" DPNH"," Reduced nicotinamide adenine dinucleotide","C00004","nicotinamide adenine dinucleotide (reduced)","CHEBI:16908","C21H29N7O14P2","DPNH","NC(=O)C1=CN(C=CC1)[C@@H]1O[C@H](COP(O)(=O)OP(O)(=O)OC[C@H]2O[C@H]([C@H](O)[C@@H]2O)n2cnc3c(N)ncnc23)[C@@H](O)[C@H]1O","1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE","Reduced nicotinamide adenine dinucleotide"]],["Database IDs",[["chebi","CHEBI:21902"],["chebi","CHEBI:13396"],["chebi","CHEBI:13395"],["chebi","CHEBI:7423"],["chebi","CHEBI:78675"],["chebi","CHEBI:57945"],["chebi","CHEBI:13392"],["chebi","CHEBI:44216"],["chebi","CHEBI:13389"],["chebi","CHEBI:16908"]]],["Standard Name","adenosine 5'-{3-[1-(3-carbamoyl-1,4-dihydropyridin-1-yl)-1,4-anhydro-D-ribitol-5-yl] dihydrogen diphosphate}"]]}},{"data":{"id":"SmallMolecule_d70ebe7ac9450bdda58b8421064a04d7_BiochemicalReaction_5747916a63a7652650dc57c8eb743bee_LEFT_TO_RIGHT","class":"simple chemical","label":"H2O","parent":"","clonemarker":true,"stateVariables":[],"unitsOfInformation":[],"bbox":{"x":727.8396,"y":693.25745,"w":45,"h":15},"parsedMetadata":[["Type","bp:SmallMolecule"],["Data Source","http://pathwaycommons.org/pc2/kegg"],["Display Name","H2O"],["Comment","REPLACED http://www.ra.cs.uni-tuebingen.de/software/KEGGtranslator/cpdC00001"],["Names",[" Water","C00001"]],["Database IDs",[["kegg reaction","R02019"],["kegg reaction","R04807"],["kegg reaction","R03220"],["kegg reaction","R08941"],["kegg reaction","R03416"],["kegg reaction","R07098"],["kegg reaction","R00022"],["kegg reaction","R02720"],["kegg reaction","R08274"],["kegg reaction","R00036"],["kegg reaction","R01718"],["kegg reaction","R07052"],["kegg reaction","R00716"],["kegg reaction","R07807"],["kegg reaction","R01710"],["kegg reaction","R00122"],["kegg reaction","R05984"],["kegg reaction","R06871"],["kegg reaction","R04826"],["kegg reaction","R04204"],["kegg reaction","R04535"],["kegg reaction","R02103"],["kegg reaction","R04746"],["kegg reaction","R02092"],["kegg reaction","R02363"],["kegg reaction","R03783"],["kegg reaction","R08725"],["kegg reaction","R00335"],["kegg reaction","R00987"],["kegg reaction","R01840"],["kegg reaction","R07082"],["kegg reaction","R00286"],["kegg reaction","R03970"],["kegg reaction","R00514"],["kegg reaction","R02973"],["kegg reaction","R04738"],["kegg reaction","R04951"],["kegg reaction","R05731"],["kegg reaction","R029
```

Comment:
Is this testfile still needed?

Reply:
No, It should have been in the .gitignore. I have removed the file now.